testcases/misc_tests/pkcsconf_test.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
```

Comment:
Is this script called during CI test anywhere? 
I would suggest to call it in make taget ci-installcheck (Makefile.am) right after calling p11sak_tests.sh.

Reply:
I will add it.

Reply:
Would it make sense to do the 2 tests in a for loop?

Reply:
for loop? I don't understand what you mean?

Reply:
maybe something like that...
```
ci-installcheck: ci-prepare installcheck
        killall -HUP pkcsslotd || true
        @sbindir@/pkcsslotd
        pushd ${srcdir}/testcases
        export PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so
        export PKCS11_USER_PIN=$(PKCS11_USER_PIN)
        for misctest in p11sak pkcsslot ; do                            \
                ./misc_tests/${misctest}_test.sh                        \
                | tee log-${misctest}.txt                               \
        done
        popd
        @sbindir@/pkcsstats --all
        killall -HUP pkcsslotd
        @echo "done"
```

Reply:
... versus 
```
ci-installcheck: ci-prepare installcheck
        killall -HUP pkcsslotd || true
        @sbindir@/pkcsslotd
        pushd ${srcdir}/testcases
        export PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so
        export PKCS11_USER_PIN=$(PKCS11_USER_PIN)
        ./misc_tests/p11sak_test.sh | tee log-p11sak.txt
        ./misc_tests/pkcsconf_test.sh | tee log-pkcsconf.txt
        done
        popd
        @sbindir@/pkcsstats --all
        killall -HUP pkcsslotd
        @echo "done"
```

Reply:
Well, with 2 tests I still think the second one is easier to read. Once we get more, then of course a for loop would make sense. Then maybe with placing the test to run in variable, like it is done in ock_tests.sh(.in).

BTW: In the second version the 'done' is superfluous.

Reply:
Ok, I'll continue with the second solution.